tests/tensorflow/sparsity/test_common.py

```diff
@@ -11,15 +11,16 @@
  limitations under the License.
 """
 
-from typing import List, Optional
+from typing import List
+from typing import Optional
 
 import pytest
 from addict import Dict
 
-from nncf.common.sparsity.schedulers import PolynomialSparsityScheduler
+from nncf.common.sparsity.schedulers import AdaptiveSparsityScheduler
 from nncf.common.sparsity.schedulers import ExponentialSparsityScheduler
 from nncf.common.sparsity.schedulers import MultiStepSparsityScheduler
-from nncf.common.sparsity.schedulers import AdaptiveSparsityScheduler
+from nncf.common.sparsity.schedulers import PolynomialSparsityScheduler
```

Comment:
Very strange changes...

Reply:
FYI, IDE can automatically reorganize imports, it's not what I am doing manually 😄